assets/stylesheets/kitten/_tokens.scss

```
@@ -4,6 +4,7 @@
 
 // Borders
 @import './tokens/borders';
+@import './tokens/borders_radius';
```

Je serais pour rester au plus proche du nom de la propriété pour les noms de ficheirs

```suggestion
@import './tokens/border-radius';
```